M2/Macaulay2/m2/chaincomplexes.m2

```diff
@@ -305,6 +308,7 @@ ChainComplexMap ++ ChainComplexMap := ChainComplexMap => (f,g) -> (
      complete g;
      scan(union(spots f, spots g), i -> h#i = f_i ++ g_i);
      h.cache.components = {f,g};
+     h.cache.formation = BinaryOperation { symbol ++, f, g };
```

Comment:
Should this be `{directSum, (f, g)}`?

Reply:
No, because the result was formed by evaluating `f ++ g`, not be evaluating `directSum(f,g)`.

Reply:
Are those different?

Reply:
Hmm -- okay, in the cases where we have code for both, I'll just replace the superfluous code for `f++g` by a call to `directSum`.